admin_guide/high_availability.adoc

```diff
@@ -349,112 +441,28 @@ ifdef::openshift-origin[]
-cluster, in that the nodes may be filtered by any constraints or requirements
-specified (e.g., nodes with SSD drives, or higher CPU, memory, or disk
-requirements, etc.).
-+
-The following example labels a highly-available cache service that is listening
-on port 9736 as *ha-cache=geo*:
-+
-====
-----
-$ oc label nodes openshift-node-{6,3,7,9} "ha-cache=geo"
-----
-====
-
-. {product-title}'s *ipfailover* internally uses *keepalived*, so ensure that
-multicast is enabled on the nodes labeled above and that the nodes can accept
-network traffic for 224.0.0.18 (the VRRP multicast IP address). Depending on
-your environment's multicast configuration, you may need to add an `iptables`
-rule to each of the above labeled nodes. If you do need to add the `iptables`
-rules, please also ensure that the rules persist after a system restart:
-+
-====
-----
-$ for node in openshift-node-{6,3,7,9}; do   ssh $node <<EOF
-export interface=${interface:-"eth0"}
-echo "Check multicast enabled ... ";
-ip addr show $interface | grep -i MULTICAST
-
-echo "Check multicast groups ... "
-ip maddr show $interface | grep 224.0.0 | grep $interface
-
-echo "Optionally, add accept rule and persist it ... "
-sudo /sbin/iptables -I INPUT -i $interface -d 224.0.0.18/32 -j ACCEPT
-
-echo "Please ensure the above rule is added on system restarts."
-
-EOF
-done;
-----
-====
-
-. Create a new *ipfailover* service account in the *default* namespace:
-+
-====
-----
-$ oc create serviceaccount ipfailover -n default
-----
-====
-
-. Add the *ipfailover* service account in the *default* namespace to the *privileged* SCC:
-+
-====
-----
-$ oadm policy add-scc-to-user privileged system:serviceaccount:default:ipfailover
-----
-====
-
-. Run a *geo-cache* service with two or more replicas. An example configuration
-for running a *geo-cache* service
-https://raw.githubusercontent.com/openshift/openshift-docs/master/admin_guide/examples/geo-cache.json[is
-provided here].
-+
-[IMPORTANT]
-====
-Be sure to replace the *myimages/geo-cache* container image referenced in the
-file with your intended image. Also, change the number of replicas to the
-desired amount and ensure the label matches the one used in the first step.
-====
 +
-----
-$ oc create -n <namespace> -f ./examples/geo-cache.json
-----
-
-. Finally, configure the VIPs and failover for the nodes labeled with
-*ha-cache=geo* in the first step. Ensure the number of replicas match the number
-of nodes and that they satisfy the label setup in first step. Specify the VIP
-addresses and the port number that *ipfailover* should monitor for the desired
-instances:
+Below is ipfailover command for the *geo-cache* service that is listening on port 9736.  Since there are 2 ipfailover deployment configration the `--vrrp-id-offset` must be set, so that each VIP gets its own offset. In this case setting a value of 10 means that the `ipf-ha-router-us-west` can have a maximum of 10 VIPs (0-9) since `ipf-ha-geo-cache`
```

Comment:
The IBM Style Guide states that we should write in words numbers that are less than 10 and use numerals for larger numbers. I always mess this up (ask @adellape)  :laughing: